website/src/contribute/release-guide.md

```diff
@@ -1135,6 +1183,20 @@ __NOTE__: Only PMC members have permissions to do it, ping [dev@](mailto:dev@bea
 
 Make sure the download address for last release version is upldaed, [example PR](https://github.com/apache/beam-site/pull/478).
 
+#### Deploy SDK docker images to DockerHub
+TODO(hannahjiang): change link to master branch after #9560 is merged.
+
+* Script: [publish_docker_images.sh](https://github.com/Hannah-Jiang/beam/blob/release_script_for_containers/release/src/main/scripts/publish_docker_images.sh)
+* Usage
+```aidl
+./beam/release/src/main/scripts/publish_docker_images.sh
+```
+Verify following points:
+* Images are published at [DockerHub](https://hub.docker.com/search?q=apachebeam&type=image) with tags :{RELEASE} and :latest.
+* Images with :latest tag are pointing to current release by confirming 
+  1. Images with :latest tags were updated a moment ago.
+  1. Size of Images with :latest tags should be same as the size of images with tag :{RELEASE}.
```

Comment:
Should `latest` and `${RELEASE}` actually are same image? If so, how about also check DIGEST.